vhosting/vhosting_check.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/python3
+import argparse
+import hashlib
+import re
+import urllib.request as req
+
+import ocflib.account.utils as utils
+import requests
+
+missing_img = 'missing_img.log'
+missing_disc = 'missing_disc.log'
+missing_both = 'missing_both.log'
+errors = 'errors.log'
+# md5 hashes
+banner_hashes = [  # ocfbadge_mini8.png
+    'b23ee41fdc537191381684dd826cf77f',
+    # ocfbadge_mini8dark.png
+    'c425451be5b2697bebe4dae12730ce8a',
+    # ocfbadge_mini8darkglow.png
+    'deaad890013a1ced05aef9ec9d023880',
+    # ocfbadge_platinum.png
+    'b1bbe54f1b85543b2e9ebb97faa43347',
+    # ocfbadge_silver8.png
+    'a69a0929a9621638dd5b94161954fa5a',
+    # ocfbadge_blue8.png
+    '0276436d44d1a5e7cfc94b0f2d96907f',
+    # binnov-157x46.gif
+    'e21bdc91cd32bc159583215168df3e14',
+    # binnov-157x46.gif.png
+    '41c167531b66bf22bfcce5f4271b337f',
+    # host-ocf.png
+    'fbd917bafa6125067ea5c0bc0bb9e912',
+    # ocfbadge_silver-Bkgd.png
+    '191d1cffd592558d6b748e2cbab2304b',
+    # ocfbanner.png
+    '46051228558531136e030bce1c45d67b',
+    # ocf.png
+    'b23ee41fdc537191381684dd826cf77f',
+    # ocfbadge_mini8dark-filtered.jpg
+    '0779ac671caee34f00887ad2222d6942',
+    # OX2Ok.png
+    '61a879bd09045605530fb5fc67b1ce9f',
+    # OCF-Flag.png
+    'ce48822dd63785f77d5826b6b215b18d',
+    # lighter152x41.gif.png
+    'a9bfe3f918552692f5eddbd3c5446367']
+# To-do: Readability
+# hosting_logos_prefix = "https://www.ocf.berkeley.edu/hosting-logos/"
+# banner_urls = [hosting_logos_prefix + banner_url for banner_url
+#                in banner_urls]
+# To-do: logs -> small DB
+disclaimer = 'We are a student group acting independently of the University'\
+             ' of California. We take full responsibility for our'\
+             ' organization and this web site.'
```

Comment:
Would it be too ugly to make this a regex which matches any whitespace rather than just spaces?